roles/command_unprivileged_verify/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Seems like we can combine the two roles and just take another parameter, e.g. `expect_failure` and have something like:

```yaml
failed_when: (ruo.rc == 0 and expect_failure) or (ruo.rc != 0 and not expect_failure)
```

?